threads.js

```diff
@@ -2036,14 +2036,32 @@ Process.prototype.reportProduct = function (a, b) {
     return +a * +b;
 };
 
+function QuotientNaNException(numerator, denominator) {
+    this.name = "Quotient exception";
+    this.message = "illegal division operation: (" + numerator + ") / (" + denominator + ")";
```

Comment:
Hmm I made a comment here but it didn't stick...
No to "Unsupported," because that sounds as if this should work but doesn't because of a Snap! deficiency.  "Undefined"?